[2024-12-24T21:12:11.207+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: monthly_collection_dag.load_data scheduled__2024-11-01T08:00:00+00:00 [queued]>
[2024-12-24T21:12:11.214+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: monthly_collection_dag.load_data scheduled__2024-11-01T08:00:00+00:00 [queued]>
[2024-12-24T21:12:11.214+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2024-12-24T21:12:11.226+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_data> on 2024-11-01 08:00:00+00:00
[2024-12-24T21:12:11.232+0000] {standard_task_runner.py:57} INFO - Started process 3434 to run task
[2024-12-24T21:12:11.235+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'monthly_collection_dag', 'load_data', 'scheduled__2024-11-01T08:00:00+00:00', '--job-id', '245', '--raw', '--subdir', 'DAGS_FOLDER/monthly_collection_dag.py', '--cfg-path', '/tmp/tmp_qpg8hmd']
[2024-12-24T21:12:11.239+0000] {standard_task_runner.py:85} INFO - Job 245: Subtask load_data
[2024-12-24T21:12:11.260+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-12-24T21:12:11.312+0000] {task_command.py:415} INFO - Running <TaskInstance: monthly_collection_dag.load_data scheduled__2024-11-01T08:00:00+00:00 [running]> on host f7bedf4d7c26
[2024-12-24T21:12:11.423+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='monthly_collection_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-01T08:00:00+00:00'
[2024-12-24T21:12:11.540+0000] {base.py:73} INFO - Using connection ID 'admin' for task execution.
[2024-12-24T21:12:11.542+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-12-24T21:12:11.554+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=monthly_collection_dag, task_id=load_data, execution_date=20241101T080000, start_date=20241224T211211, end_date=20241224T211211
[2024-12-24T21:12:11.608+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-24T21:12:11.625+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
